gensim/models/keyedvectors.py

```diff
@@ -559,6 +560,90 @@ def similar_by_vector(self, vector, topn=10, restrict_vocab=None):
         """
         return self.most_similar(positive=[vector], topn=topn, restrict_vocab=restrict_vocab)
 
+    def similarity_matrix(self, corpus, dictionary, threshold=0.0, exponent=2.0,
```

Comment:
`corpus` isn't used

Reply:
Fixed in 08dea4ebf550153b71d581fa29831674b80a9669.